packages/components/src/components/install-guide/record-instructions/IntelliJ.vue

```diff
@@ -1,65 +1,84 @@
 <template>
   <section>
-    <p class="mb20 bold">
+    <p>
       Use the <component :is="runConfigIcon" class="run-config-icon" /> "Start with AppMap" button
       to start your run configurations with AppMap enabled.
     </p>
+    <br />
     <p>
       Using "Start with AppMap" ensures that your Java code runs with the JVM option
-      <tt>-Djavaagent=appmap-agent.jar</tt>. This option is required to enable AppMap recording.
+      <code class="inline">-Djavaagent=appmap-agent.jar</code>. This option is required to enable
+      AppMap recording.
     </p>
+    <br />
+    <h2>Choose the best recording method for this project</h2>
+    <br />
     <template v-if="testFramework">
-      <h3>Tests recording</h3>
-      <p>
-        When you run your JUnit tests with the AppMap <tt>javaagent</tt> JVM argument, an AppMap
-        will be created for each test.
-      </p>
-      <p>
-        Right-click on any test class or package, and choose
-        <component :is="runConfigIcon" class="run-config-icon" /> "Start with AppMap".
-      </p>
-      <p>
-        For more information, visit
-        <a
-          href="https://appmap.io/docs/reference/jetbrains.html#create-appmaps-from-junit-test-runs"
-          target="_blank"
-          >AppMap docs - IntelliJ Tests recording</a
-        >.
-      </p>
-    </template>
-    <template v-else>
-      <VTestsPrompt frameworks="JUnit" />
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><TestsIcon /></i>Tests recording<span class="recommended-badge"
+            >recommended</span
+          >
+        </h3>
+        <p>
+          When you run your JUnit tests with the AppMap <code class="inline">javaagent</code> JVM
+          argument, an AppMap will be created for each test.
+        </p>
+        <p>
+          Right-click on any test class or package, and choose
+          <component :is="runConfigIcon" class="run-config-icon" /> "Start with AppMap".
```

Comment:
Might be too much detail, but given the prior sentence, we might need to say here "This will add the required JVM argument to enable AppMap recording."

Reply:
I think it’s useful to be explicit.

Reply:
done